horarios-react/src/components/Filter/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+
+import './style.css'
+
+import FilterButton from '../FilterButton'
+import filterImg from '../../assets/filter.svg'
+
+function Filter() {
+    const [vari, setVari] = useState(false);
+
+    function click() {
+        if (vari === "color") {
+            setVari('change-color')
+        } else {
+            setVari('color')
+        }
+    }
```

Comment:
Acho que você não está usando essa função nesse componente, verifica isso e remove se for o caso.